eras/conway/impl/src/Cardano/Ledger/Conway/CDDL.hs

```diff
@@ -0,0 +1,971 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use camelCase" #-}
+{-# HLINT ignore "Evaluate" #-}
+
+module Cardano.Ledger.Conway.CDDL where
+
+import Codec.CBOR.Cuddle.Huddle
+import Data.Function (($))
+import Data.Semigroup ((<>))
+import Data.Text qualified as T
+import Data.Word (Word64)
+import GHC.Float (Double)
+import GHC.Num (Integer)
+import GHC.Show (Show (show))
+
+default (Integer, Double)
```

Comment:
Isn't this the default default?  I think we could get rid of this :thinking: 
```suggestion
```

Reply:
Yes, it never helped anyway